src/scripts/validation.js

```diff
@@ -63,8 +62,7 @@ const activeButton = (buttonElement, config) => {
   buttonElement.classList.remove(config.inactiveButtonClass);
```

Comment:
Please, **export** the `disabledButton` function from `validation.js`, to reuse it in `index.js` for disabling


```js

export const disabledButton = (buttonElement, config) => {
  buttonElement.disabled = true;
  buttonElement.classList.add(config.inactiveButtonClass);
};
```